include/bluetooth/services/ots.h

```diff
@@ -708,6 +780,230 @@ int bt_ots_init(struct bt_ots *ots, struct bt_ots_init *ots_init);
  */
 struct bt_ots *bt_ots_free_instance_get(void);
 
+#define BT_OTS_STOP                       0
+#define BT_OTS_CONTINUE                   1
+
+/* TODO: Merge server and client instance as opaque type */
```

Comment:
Could we move the Client specific API bt_ots_client to a different file for better readability? The OTS client header can include the OTS Server header in which we can host common structures, enums etc.

This header file has over 1000 lines of code. In my opinion, having a dedicated file for the Client module would reduce the users confusion

Reply:
Well, I just spent quite a bit of work to merge the two header files, as that was requested in the PR and that is how it has been done for many of the audio services. Other files, like e.g. gatt.h are the same, I believe.
So I am reluctant to split the header file again - that would have to be as a result of some overall policy with backing from many stakeholders.